packages/jinajs/src/jinaClient.ts

```diff
@@ -48,4 +49,16 @@ export class JinaClient {
     console.log("response:", response);
     return this.serializeResponse(response.data, this.jinaVersion);
   }
+
+  async searchWithParameters(
+    documents: RawDocumentData[],
+    parameters: AnyObject
+  ): Promise<{ results: SimpleResults[]; queries: SimpleQueries }> {
+    const requestBody = await this.serializeRequest(documents, this.jinaVersion,parameters);
```

Comment:
I got rid of jinaVersion in my other PR, because it would be a better approach to solve downgrade compability with custom serializers